### PR TITLE
Feature/#10 log

### PR DESCRIPTION
**Pull request by MahdiZareie**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# jetrain
```
Comment (s-soroosh) [inline .gitignore]: jetrain?

Comment (MahdiZareie) [reply .gitignore]: yes it was raining

File: fandogh_cli/fandogh.py [hunk 2]
```diff
@@ -88,29 +82,44 @@ def publish(version):
         cleanup_workspace({})
 
 
+@click.command('build_log')
+@click.option('--version', prompt="version", help="application version")
+@login_required
+def build_log(version):
```
Comment (s-soroosh) [inline fandogh_cli/fandogh.py]: this already exists. `inspect`

**Review by s-soroosh — APPROVED** [review]

File: fandogh_cli/fandogh.py (hunk 2 above)
Comment (MahdiZareie) [reply fandogh_cli/fandogh.py]: `inspect` is not a good name IMO, shouldn't we remove `inspect` instead?

**Comment by s-soroosh** [comment]
Yes we already have an issue to rename it

**Pull request merged**